elements/dash-checkbox/dash-checkbox.js

```diff
@@ -103,6 +103,11 @@
      */
     set disabled(isDisabled) {
       this.setAttribute('aria-disabled', isDisabled);
+      if (isDisabled) {
```

Comment:
`attributeChangedCallback` only gets called for attributes listed in `observedAttributes`. It seems it only gets called if something is present so if I didn't put `disabled` on the element it would not be called.